src/plugins/vis_type_timeline/server/series_functions/opensearch/index.js

```diff
@@ -112,6 +113,14 @@ export default new Datasource('es', {
         defaultMessage: `**DO NOT USE THIS**. It's fun for debugging fit functions, but you really should use the interval picker`,
       }),
     },
+    {
+      name: 'data_source_name',
+      types: ['string', 'null'], // If null, the query will proceed with local cluster
```

Comment:
Is it possible to use optional (undefined) instead of null?

Reply:
`types` is used for the auto-complete editor and not necessarily for type annotations. Other optional arguments were listed as `['string', 'null']` as well